Formula/ncmpcpp.rb

```diff
@@ -31,6 +31,7 @@ class Ncmpcpp < Formula
   depends_on "pkg-config" => :build
   depends_on "libmpdclient"
   depends_on "readline"
+  depends_on "ncurses"
```

Comment:
macOS already provides ncurses. Why is this a dependency you think needs to be added?